niworkflows/interfaces/registration.py

```diff
@@ -88,14 +89,40 @@ class ApplyTOPUPRPT(nrc.RegistrationRC, fsl.ApplyTOPUP):
     output_spec = ApplyTOPUPOutputSpecRPT
 
     def _post_run_hook(self, runtime):
-        self._fixed_image_label = "before"
-        self._moving_image_label = "after"
-        self._fixed_image = index_img(self.inputs.in_files[0], 0)
-        self._moving_image = index_img(self.aggregate_outputs().out_corrected, 0)
-        NIWORKFLOWS_LOG.info('Report - setting before (%s) and after (%s) images',
+        self._fixed_image_label = "corrected"
+        self._moving_image_label = "warped"
```

Comment:
I have a different approach, somehow my brain links more quickly 'corrected' to "this is important" and warped to "this is the reference". Before/after requires the knowledge about what you are actually doing (I agree that's very present if you look at the final report, with the title and descriptions). Probably, the addition of the WM contour will highlight the corrected vs. warped concepts.

That said, I don't have a particular interest in the naming: I will roll back if the previous argument does not convince you.